src/main/java/it/eng/idsa/dataapp/configuration/SecurityConfig.java

```diff
@@ -13,40 +15,39 @@
 
 @Configuration
 @EnableWebSecurity
+@PropertySource("classpath:users.properties")
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
-	private static final String USERNAME = "idsUser";
-	@Value("${application.security.password}")
-	private String password;
+	@Autowired
+	private Environment env;
 
 	@Override
 	protected void configure(AuthenticationManagerBuilder auth) throws Exception {
-		auth.inMemoryAuthentication().withUser(USERNAME).password(password).roles("PROXY");
+		String usersList = env.getProperty("users.list");
+		if (usersList != null && !usersList.isEmpty()) {
```

Comment:
Can you create some UserProperties class to wrap up this logic? At least part with reading from property file (can be annotated)